funnel/templates/project_layout.html.jinja2

```diff
@@ -22,51 +22,348 @@
+                      </a>
+                    </div>
+                  {%- elif project.cfp_state.UNAVAILABLE %}
+                    <hr class="mui-divider">
+                    <div class="propose propose--lessmargin">
+                      <a class="mui-btn mui-btn--accent mui-btn--raised mui--d-block propose__btn propose__btn--strike" data-cy="Make a submission">{% trans %}Make a submission{% endtrans %}</a>
+                      <p class="mui--text-light mui--text-caption mui--text-center zero-bottom-margin">{% trans %}Submissions are closed for this project{% endtrans %}</p>
+                    </div>
+                  {%- endif %}
+                  {%- if project.primary_venue %}
+                    <p class="project-venue"><a class="mui--text-hyperlink mui--text-subhead nounderline {% if current_page == 'project' %}js-smooth-scroll{%- endif %}" href="{%- if current_page != 'project' -%}{{ project.url_for() }}{%- endif %}#venue" data-ga="View venue">{{ faicon(icon='map-marker-alt') }} <span>{{ project.primary_venue.title }}</span>{%- if project.primary_venue.city %}, <span>{{ project.primary_venue.city }}</span>{%- endif %}</a></p>
+                  {% elif project.view_for('venues').is_available() %}
+                    <a href="{{ project.url_for('venues') }}" class="mui-btn mui-btn--raised mui--d-block mui-btn--dark">{% trans %}Add venue{% endtrans %}</a>
+                  {%- endif %}
                 </div>
-                <div class="mui--text-body2 mui--text-light">{{ project.profile.description.html|firstline|truncate(150) }} <a href="{{ project.profile.url_for() }}" class="mui--text-body2">{% trans %}Read more{% endtrans %}{{ faicon(icon='arrow-right', icon_size='caption', css_class='fa-icon--left-margin') }}</a></div>
               </div>
             </div>
           </div>
-          {% block basecontentcolumninner %}{% endblock %}
         </div>
       </div>
-    {% endblock %}
+    </div>
+  </div>
+
+  {% if not project.schedule_state.PAST and project.features.tickets() -%}
+    <div class="tickets-wrapper">
+      <div class="tickets-wrapper__modal">
+        <div class="mui--hidden-md mui--hidden-lg mui--hidden-xl tickets-wrapper__modal__back mobile-nav-wrapper">
+          <div class="mobile-nav mui--z1">
+            <a href="javascript:void(0)" aria-label="{% trans %}Back to the project{% endtrans %}" class="mui--text-dark mobile-nav__icon" id="close-ticket-widget" data-ga="Back to the project page">{{ faicon(icon='arrow-left', icon_size='title') }}</a><span class="mui--text-dark mobile-nav__headline">{% trans %}Tickets{% endtrans %}</span>
+          </div>
+        </div>
+        <div id="tickets" class="tickets-wrapper__modal__body">
+          <a href="javascript:void(0)" aria-label="{% trans %}Back to the project{% endtrans %}" class="tickets-wrapper__modal__body__close" id="close-ticket-widget" data-ga="Back to the project page">{{ faicon(icon='times', baseline=false, icon_size='title') }}</a>
```

Comment:
IMO, these two need real links in the back button, not `javascript:void(0)`, for accessibility and search bots. The link can be overridden by a JS handler.